Source/WTF/wtf/ASCIICType.h

```diff
@@ -138,6 +139,11 @@ template<typename CharacterType> constexpr bool isTabOrSpace(CharacterType chara
     return character == ' ' || character == '\t';
 }
 
+template<typename CharacterType> constexpr bool isNewline(CharacterType character)
+{
+    return character == '\n' || character == '\r' || character == '\f';
+}
```

Comment:
I’m not sure that “newline” is the right name for this concept. The `'\n'` character is named newline. Just because CSS tokenization calls all three of these “newline” doesn’t mean that the right name for it here in this lower level file of fundamental character classes is `isNewline`. I think a longer name for this is probably needed to avoid ambiguity. We might even want to leave it out of `ASCIICType.h` and put it somewhere else if it might be CSS-specific.

Reply:
Yup, I've removed it from `ASCIICType.h` and now using the isNewline defined in `CSSTokenizer.cpp` when needed, as it is CSS-specific.